### PR TITLE
Get job status

### PR DESCRIPTION
**Pull request by Cooryd**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -39,6 +40,15 @@ app.post('/generate/:articleId', async (req, res) => {
         return;
     }
 });
+app.get('/status/:jobId', async (req, res) => {
+    try {
+        const status = await generationService.getJobStatus(req.params.jobId);
+        res.status(200).send(status);
+        return;
+    } catch (error) {
+        res.sendStatus(500);
```
Comment (NuclearRedeye) [inline src/index.ts:49]: Is it always a 500? What about if the jobId is not found?

**Review by NuclearRedeye — APPROVED** [review]
Just one minor comment, which if relevant can easily be addressed in a followup. Otherwise seems good to me

File: src/index.ts (hunk 1 above)
Comment (Cooryd) [reply src/index.ts:49]: At this point, without a working test environment I'm not sure what the response from Exeter would be if the job ID was incorrect. We should check that out though to return a 404 based on their response if its that an invalid request that has been made.

Comment (Cooryd) [reply src/index.ts:49]: Issue raised https://github.com/libero/editor/issues/843

**Pull request merged**